packages/resolver/lib/resolver.ts

```diff
@@ -81,14 +81,23 @@ export class Resolver {
 
   async resolve(
     importPath: string,
-    importedFrom: string
+    importedFrom: string,
+    options: {
+      compiler?: {
+        name: string;
+        version: string;
+      }
+    } = {}
```

Comment:
Should this be a typed and shared?

Reply:
My thinking was that this type was within a reasonable threshold of simplicity, since it's only defined here and in the `ResolverSource` interface. Seems a bit unnecessary for only 2 duplications?

Reply:
fair enough